bundles/org.openhab.binding.matter/NOTICE

```diff
@@ -0,0 +1,13 @@
+This content is produced and maintained by the openHAB project.
```

Comment:
I am a little surprised you are mentioning nothing in this file.

Shouldn't you mention matter.js in particular ?